web/client/plugins/Dashboards.jsx

```diff
@@ -71,7 +73,17 @@ class Dashboards extends React.Component {
             md: 2,
             className: 'ms-map-card-col'
         },
-        maps: []
+        maps: [],
+        shareOptions: {
```

Comment:
this is a clone of the one in featured maps, maybe we can use a constant pulled from the shareUtils
so in the future we will edit only in one place